data/languages/uPD77016.slaspec

```diff
@@ -205,16 +205,26 @@ attach variables [ rege  regxe  regye                                  ] [ R0E
 attach variables [ regeh regxeh regyeh                                 ] [ R0EH R1EH R2EH R3EH R4EH R5EH R6EH R7EH ];
 attach variables [                                              top_hl ] [ R0HL R1HL R2HL R3HL R4HL R5HL R6HL R7HL ];
 
-# Note: sr the following registers are labeled as unconditional and only appear in A-5.
-# A-6 gives an extra bit for dest1, but that would require 32 underscores here, so we ignore that.
-# lc is labeled as source2 only, as well.
+# Note: From disassembly it looks like the A-5 "conditional" registers are the regular pointer registers
+# and the "unconditional" registers are the system ones.  This means that bit 22 can be considered part of
+# the register (as sort of implied by the dotted line in A-5 and A-6), and it's just that system registers
```

Comment:
The dotted line is only on A-5; A-6 directly includes bit 22 without any dotted line at all. Maybe "as sort of implied by the dotted line in A-5 and the lack of a line on A-6"?

Reply:
Indeed it’s not there on the instruction layout at the top, or in the layout overview in A-1, but you can see it the dest1 layout. But perhaps that’s a copy paste from A-5?

Anyway, feel free to modify the comments as you like. I’m not precious about them in the slightest.